### ISSUE TITLE
save and load model

### ISSUE THREAD

**Issue by BerenLuthien** [CyberZHG/keras-bert#32]
**Is your feature request related to a problem? Please describe.**
When the model is saved and loaded, error happens due to "mask".
So far the workaround is saving weights and loading weights. 

Thanks

**Comment by CyberZHG** [comment]
I guess you forgot to add `MaskedGlobalMaxPool1D` to `custom_objects`.

**Comment by BerenLuthien** [comment]
I am sorry, I did typo previously. It actually complains about 
**Unknown layer: TokenEmbedding**
```
from keras.models import load_model
model = load_trained_model_from_checkpoint('bert_config.json', 'bert_model.ckpt') 
model.save('saved_model.h5')
model = load_model('saved_model.h5')
```
It is `load_model` that gives error.
Thanks for help.

**Comment by CyberZHG** [comment]
https://github.com/CyberZHG/keras-bert/blob/master/tests/test_bert.py#L20-L23
